### PR TITLE
Install/upgrade 23.04

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Install/upgrade 23.04
Preview container here: http://10.25.11.244:56567/

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [x] 23.04.x (next)

### CODE REVIEW

**Review by rsadik-ext — APPROVED** [review]

**Review by rsadik-ext — DISMISSED** [review]

File: versioned_docs/version-23.04/installation/installation-of-a-central-server/using-packages.md [hunk 1]
```diff
@@ -303,14 +320,20 @@ Install the Centreon repository using this command:
 <TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
 ```shell
-dnf install -y https://yum.centreon.com/standard/22.10/el8/stable/noarch/RPMS/centreon-release-22.10-1.el8.noarch.rpm
+dnf config-manager --add-repo https://centreon.jfrog.io/artifactory/rpm/standard/23.04/el8/centreon.repo
```
Comment (thiuyendang) [inline versioned_docs/version-23.04/installation/installation-of-a-central-server/using-packages.md:323]:
```suggestion
dnf config-manager --add-repo https://centreon.jfrog.io/artifactory/rpm-standard/23.04/el8/centreon-23.04.repo
```

File: versioned_docs/version-23.04/installation/installation-of-a-central-server/using-packages.md [hunk 2]
```diff
@@ -303,14 +320,20 @@ Install the Centreon repository using this command:
...
 ```
 
 </TabItem>
-<TabItem value="CentOS 7" label="CentOS 7">
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```shell
-yum install -y https://yum.centreon.com/standard/22.10/el7/stable/noarch/RPMS/centreon-release-22.10-1.el7.centos.noarch.rpm
+dnf config-manager --add-repo https://centreon.jfrog.io/artifactory/rpm/standard/23.04/el9/centreon.repo
+dnf config-manager --set-enabled 'centreon*'
```
Comment (thiuyendang) [inline versioned_docs/version-23.04/installation/installation-of-a-central-server/using-packages.md:334]:
```suggestion
dnf config-manager --set-enabled 'centreon*' , this command has to be deleted, only stable repo must be enabled and should be by default in the file
```

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Review by rsadik-ext — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-central-server/using-packages.md [hunk 3]
```diff
@@ -621,23 +697,25 @@ sur le serveur central :
 <TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
 ```shell
-systemctl enable php-fpm httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
+systemctl enable php-fpm httpd mariadb centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
```
Comment (rsadik-ext) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-central-server/using-packages.md:700]: systemctl enable php-fpm httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md [hunk 4]
```diff
@@ -569,23 +644,25 @@ commande suivante sur le serveur Central :
...
 systemctl start crond
 ```
 
 </TabItem>
-<TabItem value="CentOS 7" label="CentOS 7">
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```shell
-systemctl enable php-fpm httpd24-httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
+systemctl enable php-fpm httpd mariadb centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
```
Comment (rsadik-ext) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md:656]: systemctl enable php-fpm httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md [hunk 5]
```diff
@@ -569,23 +644,25 @@ commande suivante sur le serveur Central :
...
+systemctl enable crond
+systemctl start crond
 ```
 
 </TabItem>
 <TabItem value="Debian 11" label="Debian 11">
 
 ```shell
-systemctl enable php8.1-fpm apache2 centreon cbd centengine gorgoned centreontrapd snmpd snmptrapd
+systemctl enable php8.1-fpm apache2 mariadb centreon cbd centengine gorgoned centreontrapd snmpd snmptrapd
```
Comment (rsadik-ext) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md:665]: systemctl enable php-fpm httpd  centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd

File: versioned_docs/version-23.04/installation/installation-of-a-central-server/using-packages.md [hunk 6]
```diff
@@ -628,16 +704,18 @@ on the central server:
 <TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
 ```shell
-systemctl enable php-fpm httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
+systemctl enable php-fpm httpd mariadb centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
```
Comment (rsadik-ext) [inline versioned_docs/version-23.04/installation/installation-of-a-central-server/using-packages.md:707]: systemctl enable php-fpm httpd  centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd

File: versioned_docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md [hunk 7]
```diff
@@ -571,23 +651,25 @@ on the central server:
 <TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
 ```shell
-systemctl enable php-fpm httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
+systemctl enable php-fpm httpd mariadb centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
```
Comment (rsadik-ext) [inline versioned_docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md:654]: systemctl enable php-fpm httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd

File: versioned_docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md [hunk 8]
```diff
@@ -571,23 +651,25 @@ on the central server:
...
 systemctl start crond
 ```
 
 </TabItem>
-<TabItem value="CentOS 7" label="CentOS 7">
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```shell
-systemctl enable php-fpm httpd24-httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
+systemctl enable php-fpm httpd mariadb centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd
```
Comment (rsadik-ext) [inline versioned_docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md:663]: systemctl enable php-fpm httpd centreon cbd centengine gorgoned snmptrapd centreontrapd snmpd

File: versioned_docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md [hunk 9]
```diff
@@ -571,23 +651,25 @@ on the central server:
...
+systemctl enable crond
+systemctl start crond
 ```
 
 </TabItem>
 <TabItem value="Debian 11" label="Debian 11">
 
 ```shell
-systemctl enable php8.1-fpm apache2 centreon cbd centengine gorgoned centreontrapd snmpd snmptrapd
+systemctl enable php8.1-fpm apache2 mariadb centreon cbd centengine gorgoned centreontrapd snmpd snmptrapd
```
Comment (rsadik-ext) [inline versioned_docs/version-23.04/installation/installation-of-a-remote-server/using-packages.md:672]: systemctl enable php8.1-fpm apache2 centreon cbd centengine gorgoned centreontrapd snmpd snmptrapd

**Review by rsadik-ext — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-central-server/using-packages.md [hunk 10]
```diff
@@ -205,30 +232,57 @@ dnf module install php:remi-8.1
...
-#### Dépôt Remi
+</TabItem>
+<TabItem value="RHEL 9" label="RHEL 9">
 
-Pour installer Centreon, vous devez installer le dépôt **remi**.
+Exécutez les commandes suivantes :
+
+```shell
+dnf install dnf-plugins-core
+dnf install epel-release
```
Comment (rsadik-ext) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-central-server/using-packages.md:259]:
```suggestion
dnf install -y http://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-central-server/using-packages.md [hunk 11]
```diff
@@ -205,30 +232,57 @@ dnf module install php:remi-8.1
...
+subscription-manager repos --enable codeready-builder-for-rhel-9-x86_64-rpms
+```
```
Comment (rsadik-ext) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/installation/installation-of-a-central-server/using-packages.md:261]:
```suggestion
dnf config-manager --set-enabled codeready-builder-for-rhel-9-rhui-rpms
```
```

**Review by rsadik-ext — CHANGES_REQUESTED** [review]
You can do these changes for redhat9 for central & remote server too

**Review by thiuyendang — APPROVED** [review]

**Review by rsadik-ext — APPROVED** [review]

**Pull request merged**